ios/FluentUI/Avatar/AvatarView.swift

```diff
@@ -587,8 +592,10 @@ open class AvatarView: UIView {
     private func updateBorderColor() {
         if let borderColor = borderColor {
             borderView.backgroundColor = borderColor
-        } else {
+        } else if (primaryText ?? "").isEmpty && (secondaryText ?? "").isEmpty {
             borderView.backgroundColor = Colors.Avatar.border
+        } else {
+            borderView.backgroundColor = initialsView.backgroundColor
```

Comment:
>initialsView [](http://example.com/codeflow?start=41&length=12)

is this a new design direction that even if we have an image we will try to follow the intialsView background color for border?

Reply:
For context: the Vnext Avatar sets the background color as the ring color by default if the style is not "overflow" and there are strings to allow the calculation of the background color.

Each Avatar Vnext style defines its own default ring color.

To prevent breaking the current behavior here, how about creating a property like "shouldCalculateRingColor" which defaults to false keeping the default border color with the gray unless the property is set to true by the client code?